tests/src/python/test_provider_spatialite.py

```diff
@@ -1561,8 +1561,10 @@ def testSpatialiteDefaultValues(self):
 
         # Test default values
         dp = vl.dataProvider()
-        # FIXME: should it be None?
-        self.assertTrue(dp.defaultValue(0).isNull())
+        if int(QT_VERSION_STR.split('.')[0]) >= 6:
```

Comment:
I think this test is actually wrong -- it should be

     self.assertEqual(dp.defaultValue(0), NULL)

That should work in both qt 5 and 6.

Reply:
confirmed, done

Reply:
@rouault looks like there's a real bug here -- https://github.com/qgis/QGIS/pull/56578